core/transaction_pool/clock.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CLOCK_HPP
+#define KAGOME_CLOCK_HPP
+
+#include <chrono>
+
+namespace kagome::transaction_pool {
+
+  /**
+   * An interface for a clock
+   */
+  class Clock {
```

Comment:
also there should be some requirement for the clock, if it is a steady clock or high precision clock or something else.

If you don't know requirements, just use steady clock.
https://en.cppreference.com/w/cpp/chrono/steady_clock